src/disclosures/css/disclosures.less

```diff
@@ -50,6 +69,191 @@
 }
 
 /* ==========================================================================
+   Review your first year offer
+   ========================================================================== */
+
+.offer-part {
+  margin-top: unit(45px / @base-font-size-px, em);
+
+  .form-group {
+    margin-top: unit(45px / @base-font-size-px, em);
+  }
+
+  .form-label-header {
+    padding-bottom: unit(5px / @small-font-size-px, em);
+    border-bottom: 1px solid @gray-50;
+    margin-bottom: unit(15px / @small-font-size-px, em);
+  }
+
+  .content_line {
+    margin-top: unit(15px / @base-font-size-px, em);
+    margin-bottom: unit(7px / @base-font-size-px, em);
+  }
+}
+
+.offer-part_heading {
+  .heading-3();
+
+  .respond-to-min(@bp-sm-min, {
+    .heading-2();
+  });
+
+  & + .form-group {
+    margin-top: unit(30px / @base-font-size-px, em);
+  }
+}
+
+.offer-part_unit {
+  line-height: 2;
+}
+
+.offer-part_definition {
+  font-size: unit(14px / @base-font-size-px, em);
+}
+
+.form-label + .offer-part_definition {
+  margin-top: unit(-5px / @small-font-size-px, em);
+}
+
+.offer-part_currency {
+  position: relative;
+
+  .offer-part_unit {
+    position: absolute;
+    right: unit(58px / @base-font-size-px, em);
+  }
+
+  .offer-part_input {
+    box-sizing: border-box;
+    width: 100%;
+    text-align: right;
+  }
+}
+
+.offer-part_non-currency {
+
+  .offer-part_input {
+    box-sizing: border-box;
+    width: unit(40px / @base-font-size-px, em);
+  }
+
+  .offer-part_unit {
+    display: inline-block;
+    margin-left: unit(10px / @base-font-size-px, em);
+  }
+}
+
+.offer-part_summary {
+  padding-bottom: unit(30px / @base-font-size-px, em);
+  margin-top: unit(45px / @base-font-size-px, em);
+  margin-bottom: unit(45px / @base-font-size-px, em);
+  background-color: @gold-20;
+
+  &.debt-summary {
+    margin-top: 0;
+    background-color: @gold-50;
+  }
+
+  .content_line {
+    margin-top: unit(15px / @base-font-size-px, em);
+    margin-bottom: unit(7px / @base-font-size-px, em);
+  }
+}
+
+.offer-part_summary-heading {
+  .heading-5();
+  border-bottom: 1px solid @black;
+  margin-bottom: unit(10px / @font-size, em);
+}
+
+// Elements need to be nested here to inherit @font-size
+.line-item {
+  .grid_nested-col-group();
+  .heading-4();
+  margin-bottom: unit(15px / @font-size, em);
+
+  &_sign {
+    position: relative;
+    left: unit(-5px / @font-size, em);
+    font-size: unit(18px / @font-size, em);
+  }
+
+  &_currency {
+    position: absolute;
+    right: unit(77px / @font-size, em);
+  }
+}
+
+.line-item_title {
+  .grid_column(7);
+}
+
+.line-item_value {
+  .grid_column(5);
+}
+
+.line-item_amount {
+  float: right;
```

Comment:
Good question. I tried this a few ways, and this was the only way I could make all these constrains happen:

- Label is always flush left
- Number is always flush right
- Dollar sign sits 5 digits plus one comma away from the right edge
- Plus or minus sign sits just to the left of the dollar sign

The trouble I kept running into was the the dollar, plus, and minus signs don't quite fit to the grid. I have a feeling this might need to be adjusted to work in larger screen sizes anyway, so I'll keep noodling on it.